test/app/breadcrumb_maker_test.js

```diff
@@ -0,0 +1,57 @@
+var assert = require('assert');
```

Comment:
OK, based on this comment I have changed our testing framework to Jasmine. Thankfully it was a pretty small change and our existing tests continue to work with minimal changes.